src/browsers/gp2/GP2Browser.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react'
+
+import Browser from '../base/Browser'
+import { renderCount } from '../base/tableCells'
+
+import GP2AboutPage from './GP2AboutPage'
+import GP2HomePage from './GP2HomePage'
+import GP2VariantFilter from './GP2VariantFilter'
+
+import vepConsequences from '../base/vepConsequences'
+
+const variantConsequences = [...vepConsequences]
+
+const renderOddsRatio = (value) => {
+  if (value === null) {
+    return ''
+  }
+  if (value === 'Infinity') {
+    return '∞'
+  }
+  if (value === 0) {
+    return '0'
+  }
+  return value.toPrecision(3)
+}
+
+const GP2Browser = () => (
+  <Browser
+    browerTitle="GP2 Browser"
+    navBarBackgroundColor="#7b558c"
+    homePage={GP2HomePage}
+    extraPages={[
+      {
+        path: '/about',
+        label: 'About',
+        component: GP2AboutPage,
+      },
+    ]}
+    geneResultsPageHeading="GP2: gene burden results"
+    // TODO: change this, these are b/c we're using Epi25 gene results rn
+    //
+    geneResultsAnalysisGroupOptions={[
```

Comment:
let's pull these parameters out into separate `const`s to de-chonk this component